### PR TITLE
array-object-function-exercises

### PR DESCRIPTION
**Pull request by rahelbelay**

### CODE REVIEW

File: scripts/index.js [hunk 1]
```diff
@@ -1 +1,244 @@
-// Your code goes here!
\ No newline at end of file
+// Your code goes here!
+const wholeNumbers = [-20, 10, 15, -1, 30, 0, 3, -3];
+ 
+// positive numbers
+
+function isPositve(array) {
+    newArray = [];
```
Comment (radishmouse) [inline scripts/index.js:7]: Don't forget to use `let` or `const`!

File: scripts/index.js [hunk 2]
```diff
@@ -1 +1,244 @@
...
+
+// Str Multiply
+
+
+function strMultiply (str, times) {
+    const newString = [];
+    for(let i = 0; i < times; i++) {
+        newString.push(str)   
+    }
+    console.log(newString.join("")) ;
```
Comment (radishmouse) [inline scripts/index.js:182]: Great! Good technique for working with an array and converting to a string 👍

**Review by radishmouse — COMMENTED** [review]
Great work!
Try to remember your semi-colons at the end of each statement!
😸

**Comment by rahelbelay** [comment]
thank you!!!

**Pull request closed without merging**